webextension/scripts/background.js

```diff
@@ -344,7 +344,7 @@ function getCachedFactCheck(url, onSuccess, onFail) {
 chrome.runtime.onStartup.addListener((details) => {
   chrome.storage.local.get({ agreement: false }, (settings) => {
     if (settings && settings.agreement) {
-      chrome.browserAction.setPopup({ popup: chrome.runtime.getURL('index.html') })
+      chrome.browserAction.setPopup({ popup: chrome.runtime.getURL('index.html') },callback)
```

Comment:
Would be great to leave a space after the comma, as per the remainder of this codebase.